Add GeoTIFF and BBox validation nodes

rmarquis opened the pull request:
- **feat(api): implement GeoTIFF metadata reader node for 3D tiles workflow**
- **chore(api): add @cf-wasm/png dependency**
- **feat(api): add GeoTIFF query and BBox validation nodes**

bchapuis commented on apps/api/src/nodes/3d/types.ts:
```diff
@@ -0,0 +1,18 @@
+/** GeoTIFF metadata structure for JSON output */
+export interface GeoTiffMetadata {
```
I think we should move this in the GeoTiffMetadataReader node and keep it private.

bchapuis commented on pnpm-lock.yaml:
```diff
@@ -10568,6 +10621,8 @@ snapshots:
 
   leac@0.6.0: {}
 
+  lerc@3.0.0: {}
```
Why do we need these dependencies? Can some of them be removed?

bchapuis reviewed (changes requested):
I added a couple of comments in the pull request.

rmarquis commented:
* Bbox validation is now integrated to the GeoTIFFQuery node. It's also now optional, if the param is not passed, then the node takes the full boudaries of the GeoTIFF.
* removed ghost dependencies
* adjusted code to hide complexity deeper
